Dockerfile-debian.template

```diff
@@ -0,0 +1,53 @@
+FROM debian:buster-slim
+
+ENV DART_VERSION   {{DART_VERSION}}
+ENV CHANNEL        {{CHANNEL}}
+ENV PLATFORM       {{PLATFORM}}
+ENV ARCH           {{ARCH}}
+
+ENV DART_SDK /usr/lib/dart
+
+RUN set -eux; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends \
+		ca-certificates \
+		curl \
+		openssh-client \
+		unzip \
+		dnsutils \
+	; \
+	rm -rf /var/lib/apt/lists/*
+
+WORKDIR /root
+RUN set -eux; \
+    SDK="dartsdk-$PLATFORM-$ARCH-release.zip"; \
+    BASEURL="https://storage.googleapis.com/dart-archive/channels"; \
+    URL="$BASEURL/$CHANNEL/release/$DART_VERSION/sdk/$SDK"; \
+    echo "SDK: $URL" >> dart_setup.log ; \
+    curl -fLO "$URL" && curl -fLO "$URL".sha256sum; \
+    sha256sum --status -c "$SDK".sha256sum; \
+    unzip "$SDK" && mv dart-sdk "$DART_SDK" && rm "$SDK";
+
+ENV PATH $DART_SDK/bin:$PATH
+
+# create a minimal "scratch" environment for creating hyper-minimal images
+# usage: COPY --from=dart:xxx /scratch-runtime/ /
+# uses hard links here to save space
+RUN set -eux; \
+	for f in \
```

Comment:
Should we also copy in https://dart.dev/tools/dartaotruntime?

Then the app being served could be even smaller [AOT snapshots](https://dart.dev/tools/dart-compile#aot-snapshot) rather than a full exe.

WDYT @jonasfj ?

Reply:
Moved to https://github.com/dart-lang/dart-docker/issues/3